test/jdk/java/awt/Cursor/CursorDragTest/ListDragCursor.java

```diff
@@ -78,7 +76,61 @@ public static Frame createUI() {
         panel.add(list);
 
         frame.add(panel);
-        frame.setBounds(300, 100, 300, 150);
-        return frame;
+        frame.setSize(300, 150);
+        frame.setLocation(450, 400);
```

Comment:
Allows more space between the windows

```suggestion
        frame.setLocation(450, 500);
```